tema1/schedule.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Schedule</title>
+</head>
+
+<body>
+  <header>
+    <h1>My schedule</h1>
+    <label for="time">April 2024</label>
+    <meter id="time" value="0.7" style="width:70%"">Percentage</meter><br>
+    <nav>
+      <a href="/index.html" target="_blank">Home</a> |
+      <a href="/bio.html" target="_blank">Bio</a> |
+      <a href="#" target="_blank">Schedule</a>
+    </nav>
+  </header>
+  <main>
+    <table border="2" style="width:70%">
+      <thead>
+        <tr>
+          <th></th>
+          <th>
+            <h4>Monday</h4>
+          </th>
+          <th>
+            <h4>Tuesday</h4>
+          </th>
+          <th>
+            <h4>Wednesday</h4>
+          </th>
+          <th>
+            <h4>Thursday</h4>
+          </th>
+          <th>
+            <h4>Friday</h4>
+          </th>
+          <th>
+            <h4>Saturday</h4>
+          </th>
+          <th>
+            <h4>Sunday</h4>
+          </th>
+        </tr>
+      <tbody>
+        <tr>
+          <td width>
```

Comment:
Nu cred ca trebuie un width neaparat aici
```suggestion
          <td>
```

Reply:
voisem sa incerc nu stiu ce chestie pe atunci..:|